tests/integration/utils.go

```diff
@@ -1224,14 +1224,13 @@ func createDoc(
 	}
 	s.docIDs[action.CollectionID] = append(s.docIDs[action.CollectionID], docIDs...)
 
+	docIDMap := make(map[string]struct{})
+	for _, docID := range docIDs {
+		docIDMap[docID.String()] = struct{}{}
+	}
+
 	if action.ExpectedError == "" {
-		waitForUpdateEvents(
-			s,
-			action.NodeID,
-			action.CollectionID,
-			getEventsForCreateDoc(s, action),
-			action.Identity,
-		)
+		waitForUpdateEvents(s, action.NodeID, action.CollectionID, docIDMap, action.Identity)
```

Comment:
question: curious why this needed to be done?

Reply:
Because the ids were already available on create and since the embedding vector is not know in advance, we cannot depend on the documents content in the test's definition to determine the docIDs.